site/src/pages/WorkspacesPage/WorkspacesPageView.tsx

```diff
@@ -65,7 +65,7 @@ export interface WorkspacesPageViewProps {
   canChangeVersions: boolean;
 }
 
-export const WorkspacesPageView = ({
+export const WorkspacesPageView: FC<WorkspacesPageViewProps> = ({
```

Comment:
Any reason you explicitly want to use FC here? or is change to make this more consistent with other components?

Reply:
just consistency